src/puppetlabs/puppetdb/cli/benchmark.clj

```diff
@@ -375,89 +380,41 @@
           (if (> time-diff 5000)
             (let [time-diff-seconds (/ time-diff 1000)
                   messages-per-second (float (/ events-since-last-report time-diff-seconds))]
-              (println "Sending" messages-per-second "messages/s"
-                       "(load equivalent to" (int (/ messages-per-second expected-node-message-rate)) "nodes)")
-              (flush)
+              (println-err
+               (trs
+                "Sending {0} messages/s (load equivalent to {1} nodes)"
+                messages-per-second
+                (int (/ messages-per-second expected-node-message-rate))))
+              (binding [*out* *err*] (flush))
```

Comment:
Good question - just kept the flush from before; perhaps so, assuming it respects *flush-on-newline*.